themes/bootstrap3/templates/RecordDriver/DefaultRecord/versions-link.phtml

```diff
@@ -0,0 +1,21 @@
+<?php
+  $versions = $this->driver->tryMethod('getOtherVersionCount');
+  if ($versions):
```

Comment:
You can `return` from a template to stop rendering; maybe it's cleaner to say:

<pre>
if (!$versions) {
    return;
}
</pre>

...and then it's not necessary to have nested php blocks or to indent the rest of the template.

Reply:
Changed.